mozilla_aws_cli/static/index.js

```diff
@@ -7,6 +7,8 @@ const state = {
     getStateCount: 0,
     lastRole: undefined,
     roleRetrievalCount: 0,
+    backendInProgress: false,
```

Comment:
Alphabetical order?

Reply:
Done in 4c19b5d43a469e4c2fccef8538be1c2f00c7cb31